app/src/main/java/org/mozilla/focus/widget/Cursor.java

```diff
@@ -24,7 +24,7 @@
 public class Cursor extends View {
 
     private float CURSOR_SIZE = 45;
-    private final int MAX_SPEED = 15;
+    private final int MAX_SPEED = 25;
```

Comment:
I actually meant the initial presses felt slow (is it just the remote latency though?) but that being said, this still feels a little better, I think.